services/modules/slack/manager.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import unicode_literals
+import random
+import string
+import requests
+from eveonline.managers import EveManager
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from slackclient import SlackClient
+
+from six import iteritems
+from .tasks import SlackTasks
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlackManager:
+    def __init__(self):
+        pass
+
+    @classmethod
+    def _response_ok(cls, response):
+        try:
+            if response['ok']:
+                return True
+            else:
+                return False
+        except:
+            return False
+
+    @staticmethod
+    def exec_request(endpoint, func, email, **kwargs):
+        try:
+            sc = SlackClient(settings.SLACK_TOKEN)
+            res = sc.api_call('users.admin.invite', email=email, resend=True)
+            return res
+        except:
+            logger.exception("Error encountered while performing API request to Slack.")
+            return {}
+
+    @classmethod
+    def invite_user(cls, request):
```

Comment:
I would avoid passing in the entire request object and pass in just the user instead. Nothing else from the request object is used or needed so there's not really any reason to require it be passed like this. This way if someone so desires they could write a script to invite all their users.